graph/collection-properties.md

```diff
@@ -0,0 +1,402 @@
+OData services treat collections of complex types differently than they treat collections of entity types. 
+This is due to the nature of entity types being "individually addressable" (they have some key which uniquely identifies them within the collection) while complex types are not individually addressable. 
+The result of this difference is that it is almost always preferable to use entity types for collections rather than complex types; please see the exceptions [here](#exceptions).
+Let's use the following CSDL as an example:
+
+```xml
+<EntityType Name="interestingData">
+  <Property Name="foos" Type="Collection(self.foo)" Nullable="false" />
+  <NavitationProperty Name="bars" Type="Collection(self.bar)" Nullable="false" ContainsTarget="true" />
+</EntityType>
+
+<ComplexType Name="foo">
+  <Property Name="someProperty" Type="Edm.String" Nullable="false" />
+</ComplexType>
+
+<EntityType Name="bar">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
+  <Property Name="differentProperty" Type="Edm.Int32" Nullable="false" />
+</EntityType>
+```
+
+## Retrieving the elements in a collection
+
+The [OData](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#_Toc31358935) [standard](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#_Toc31358947) specifies `foos` and `bars` both can be retrieved using a `GET` request:
+> OData services support requests for data via HTTP GET requests.
+> 
+> ...
+> 
+> OData services support querying collections of entities, complex type instances, and primitive values.
+
+### Complex Type
```

Comment:
As i understood your suggestion was to prefer collection of entity types when there is a need to update individual elements therefore i would remove references to complex types and show difference only when it's absolutely essential so that the patter would be relatively short. IMO if you show differences for GET then the POST doesn't add much value. While PATCH is important.